packages/smart-contracts/package.json

```diff
@@ -78,7 +78,7 @@
     "@types/mocha": "8.2.3",
     "@types/node": "18.11.9",
     "chai": "4.3.4",
-    "dotenv": "10.0.0",
+    "dotenv": "16.5.0",
```

Comment:
Even if not broken, let's use the same version of `dotenv` as in the other packages